ConfigSpace/forbidden.py

```diff
@@ -182,8 +182,8 @@ def __init__(self, *args):
     @abstractmethod
     def __repr__(self):
         pass
-
-    def get_descendant_literal_clauses(self):
+    # todo: recheck is return type should be AbstractForbiddenComponent or AbstractForbiddenConjunction or Hyperparameter
```

Comment:
mypy accepts only List[hyperparameter] but according to debugger AbstractForbiddenComponent should also work